docs/Tutorials/MathComponent/md/Tutorial.md

```diff
@@ -1052,7 +1051,7 @@ _(v1 + v2) f_.
 the event throttle.
 
 1. **Telemetry:**
-There two telemetry channels: one for reporting
+There are two telemetry channels - one for reporting
```

Comment:
A hyphen is not correct here. The original colon is correct. The correct alternative punctuation is an em-dash.